readthedocs/proxito/exceptions.py

```diff
@@ -0,0 +1,174 @@
+from django.http import Http404
+from django.utils.translation import pgettext_lazy
+
+
+class ContextualizedHttp404(Http404):
+
+    """
+    Base class for contextualized HTTP 404 handling.
+
+    Subclasses may define their own template name,
+    HTTP status and object that was not found.
+
+    The contextualized exception is handled by proxito's 404 handler
+    """
+
+    template_name = "errors/404/base.html"
+    not_found_subject = pgettext_lazy(
+        "Names an object not found in a 404 error", "page"
+    )
```

Comment:
This is being used in the templates as

```
The {{ not_found_subject }} you are looking for at <code>{{ path_not_found }}</code> was not found.
```

Not sure if other languages require the word to be translated as a whole with the sentence, @agjohnson may have some input here.

Reply:
Added better translator context guidelines in 5e9017e69781214296ad807e2f6aae4433e2a693

I can imagine several ways to improve things, also for translators - but I don't think this will lead to inaccuracy, now that there's also a pretty solid context.

I don't think it's proportional to put more efforts on the translation of this particular part, given that there hasn't been much effort in designing the error messages (yet).